src/main/java/org/reactome/addlinks/fileprocessors/VGNCFileProcessor.java

```diff
@@ -0,0 +1,61 @@
+package org.reactome.addlinks.fileprocessors;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class VGNCFileProcessor extends FileProcessor{
+
+    public VGNCFileProcessor(String processorName)
+    {
+        super(processorName);
+    }
+
+    public VGNCFileProcessor()
+    {
+        super(null);
+    }
+
+    @Override
+    public Map<String, List<String>> getIdMappingsFromFile()
+    {
+        Map<String, List<String>> mappings = new HashMap<>();
+        Path inputFile = Paths.get(this.pathToFile.toAbsolutePath().toString().replace(".gz", ""));
+        try
+        {
+            this.unzipFile(this.pathToFile, true);
+
+            for (String line : Files.readAllLines(inputFile).stream().skip(1).collect(Collectors.toList())) {
+                List<String> tabSplit = Arrays.asList(line.split("\t"));
+                if (tabSplit.size() > 21) {
+                    String vgncId = tabSplit.get(1).split(":")[1];
+                    String ensemblId = tabSplit.get(20);
+                    List<String> uniprotIds = Arrays.asList(tabSplit.get(21).split("\\|"));
```

Comment:
Could we get a little note on the significance of 20 and 21 in terms of the size of the `tabSplit` list?